build-configs/configs/malte/index.js

```diff
@@ -28,7 +28,8 @@ const MalteBuildConfig: BuildConfigType = {
     }
   },
   ios: {
-    bundleIdentifier: 'de.integreat-app',
+    bundleIdentifier: 'de.malteapp',
+    provisioningProfileSpecifier: 'match Development de.malteapp',
```

Comment:
Do we really need this? Wouldn't it be enough to just use `match Development ${bundleIdentifier}`?

Reply:
I'd try to avoid having redundant information here if possible.

Reply:
This option makes sense if you want to use a AdHoc or AppStore profile. Then you can quickly set this in the build config.